### PR TITLE
Enhancement/86eq5k2ud add tooltip on simulator header

### PR DESCRIPTION
**Pull request by nasolgabriel**
added tooltip to simulator header options

image
image

### CODE REVIEW

File: apps/simulator/src/components/Toolbar/Toolbar.tsx [hunk 1]
```diff
@@ -30,23 +31,30 @@ export const ToolbarSettings: React.FC = () => {
     },
   };
 
+  const toolTipData = [
```
Comment (DropitupQwerty) [inline apps/simulator/src/components/Toolbar/Toolbar.tsx:34]: Can you seperate this in our constant?

File: apps/simulator/src/components/Header/Header.tsx [hunk 2]
```diff
@@ -82,15 +83,15 @@ export const Header: React.FC<Props> = ({ logout, current }) => {
               </Box>
               <Box flexGrow={0}>
                 <div className="header-step-5">
-                  <Tooltip title="current page">
+                  <CustomTooltip title="current page">
                     <Button sx={{ color: 'white', fontSize: 'small' }}>
                       <AutoStoriesIcon
                         fontSize="small"
                         style={{ fontFamily: 'Arial, sans-serif', marginRight: '2px' }}
                       />{' '}
```
Comment (DropitupQwerty) [inline apps/simulator/src/components/Header/Header.tsx:91]: Do you use this blank space as space?

File: apps/simulator/src/components/Toolbar/Toolbar.tsx (hunk 1 above)
Comment (DropitupQwerty) [inline apps/simulator/src/components/Toolbar/Toolbar.tsx:34]: Can you isolate this in a constant folder?

**Review by DropitupQwerty — CHANGES_REQUESTED** [review]

**Review by JMSevilla — CHANGES_REQUESTED** [review]
Please address existing comments

File: apps/simulator/src/components/Toolbar/Toolbar.tsx [hunk 3]
```diff
@@ -30,23 +31,30 @@ export const ToolbarSettings: React.FC = () => {
...
-            <Button sx={iconStyle} onClick={handleZoomInText}>
-              <ZoomInIcon fontSize="medium" />
-            </Button>
+            {toolTipData.map((item,index)=>(
+              <CustomTooltip title={item.title}>
+                <Button sx={iconStyle}>
+                  {item.icon} 
+                </Button>
+            </CustomTooltip>
+            ))}
```
Comment (be3jaay) [inline apps/simulator/src/components/Toolbar/Toolbar.tsx:56]: You didn't used the index, I believed there will be a warning on console when you didn't put a unique "key" prop in a list, 
```suggestion
            {toolTipData.map((item,index)=>(
              <CustomTooltip key={index} title={item.title}>
                <Button sx={iconStyle}>
                  {item.icon} 
                </Button>
            </CustomTooltip>
            ))}
```

**Review by be3jaay — CHANGES_REQUESTED** [review]

File: apps/simulator/src/core/constant/toolTipDataConstant.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    border: '1px solid #f3f3f3',
+    margin: 1,
+    backgroundColor: '#007AB7',
+    color: '#030303',
+    '&:hover': {
+      color: '#f3f3f3',
+    },
+  };
+
+export const ToolTipData = () => {
```
Comment (be3jaay) [inline apps/simulator/src/core/constant/toolTipDataConstant.tsx:17]: Your file name should match the name of the component. Also, toolTipDataConstant is too long (TooltipData is enough).

**Review by be3jaay — CHANGES_REQUESTED** [review]

File: apps/simulator/src/core/constant/toolTipDataConstant.tsx (hunk 4 above)
Comment (nasolgabriel) [reply apps/simulator/src/core/constant/toolTipDataConstant.tsx]: toolTipDataConstant already renamed

**Review by be3jaay — APPROVED** [review]

**Review by JMSevilla — APPROVED** [review]

**Review by LizterBabael — APPROVED** [review]

**Review by DropitupQwerty — APPROVED** [review]

**Pull request merged**